docs/usage/faq.md

```diff
@@ -29,10 +29,12 @@ The Renovate team only fixes bugs in an older version if:
 If you're using the hosted app, you don't need to do anything, as the Renovate maintainers update the hosted app regularly.
 If you're self hosting Renovate, use the latest release if possible.
 
-## When are new Renovate OSS releases added to the hosted Mend Renovate App?
+## When is Renovate's GitHub hosted app updated?
```

Comment:
Am I allowed to use the `'` character in the heading? I suspect this `'s` thing will break something in the docs or in the tests.

Reply:
you can always flip it with "of"
```suggestion
## When is the GitHub hosted app of Renovate updated?
```